stock_vertical_lift_empty_tray_check/views/res_config_setting_views.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+        <record id="res_config_settings_view_form" model="ir.ui.view">
+            <field name="name">res.config.settings.view.form.inherit.stock</field>
+            <field name="model">res.config.settings</field>
+            <field name="priority" eval="30" />
+            <field name="inherit_id" ref="stock.res_config_settings_view_form" />
+            <field name="arch" type="xml">
+                <xpath
+                    expr="//div[descendant-or-self::field[@name='group_stock_adv_location'] and @class='col-12 col-lg-6 o_setting_box']"
```

Comment:
I missed this... this select seems weird. What are you trying to select? Also, it causes
`021-03-12 10:15:17,541 185 WARNING openerp_test odoo.addons.base.models.ir_ui_view: Error-prone use of @class in view res.config.settings.view.form.inherit.stock (): use the hasclass(*classes) function to filter elements by their classes`

Reply:
as discussed: use ancestor to get to the closest setting_box

Reply:
Changed by ```//field[@name='group_stock_adv_location']/ancestor::div[hasclass('o_setting_box')]``` as you suggested. Thanks!